src/Symfony/Component/EventDispatcher/EventDispatcherInterface.php

```diff
@@ -23,15 +23,15 @@
     /**
      * Dispatches an event to all registered listeners.
      *
-     * @param string $eventName The name of the event to dispatch. The name of
-     *                          the event is the name of the method that is
-     *                          invoked on listeners.
-     * @param Event  $event     The event to pass to the event handlers/listeners
-     *                          If not supplied, an empty Event instance is created.
+     * @param string          $eventName The name of the event to dispatch. The name of
+     *                                   the event is the name of the method that is
+     *                                   invoked on listeners.
+     * @param EventInterface  $event     The event to pass to the event handlers/listeners
+     *                                   If not supplied, an empty Event instance is created.
      *
-     * @return Event
+     * @return EventInterface
      */
-    public function dispatch($eventName, Event $event = null);
+    public function dispatch($eventName, EventInterface $event = null);
```

Comment:
Every class implementing this interface must be updated too (see the fatal error you will get otherwise: https://3v4l.org/CQgFB).